core/sr-io/Cargo.toml

```diff
@@ -33,7 +33,9 @@ std = [
 	"environmental",
 	"substrate-state-machine",
 	"libsecp256k1",
-	"tiny-keccak"
+	"tiny-keccak",
+	"no_oom",
+	"no_panic_handler",
```

Comment:
Why do we need those? I thought those are features required for SGX buids.

Reply:
It’s to prevent duplicate lang items.

Reply:
I think that is not right, the proper definitions should come from `sr-io` not from elsewhere, in the case we are compiling for `no_std` (i.e the wasm runtime)

AFAIU, these features are needed for the case where a runtime is compiled for `no_std` but for SGX not wasm and SGX needs to provide its own definitions of these handlers.